### ISSUE TITLE
[NFR] PHQL: Namespace aliases

### ISSUE THREAD

**Issue by golovanov** [phalcon/cphalcon#657]
If we use namespaces with company name, project name, module name, etc.. then we will get a long-long query :(

Example:

```php
$phql = "
    SELECT c.*, b.*
    FROM MyCompanyName\MyProjectName\MyStoreModule\Models\Cars AS c
    INNER JOIN VendorCompanyName\VendorProjectName\VendorModuleName\Models\Brands AS b ON b.id = c.brands_id
    ";
```

I suggest to make use aliases for namespaces.
p.s. aliases can be uses for modules, vendors, etc.
```php
$di->modelsManager->registerQueryNamespaceAlias('Store', 'MyCompanyName\MyProjectName\MyStoreModule\Models');
$di->modelsManager->registerQueryNamespaceAlias('BrandLibrary', 'VendorCompanyName\VendorProjectName\VendorModuleName\Models');

$phql = "
    SELECT c.*, b.*
    FROM Store::Cars AS c
    INNER JOIN BrandLibrary::Brands AS b ON b.id = c.brands_id
    ";

```

**Comment by tmihalik** [comment]
+1

**Comment by andresgutierrez** [comment]
This implemented in 2.0.3

**Comment by andresgutierrez** [comment]
```php
<?php

namespace Some;

use Phalcon\DI,
    Phalcon\Db\Column,
    Phalcon\Mvc\Model,
    Phalcon\Events\Manager as EventsManager,
    Phalcon\Db\Adapter\Pdo\MySQL as Connection,
    Phalcon\Mvc\Model\Manager as ModelsManager,
    Phalcon\Mvc\Model\Metadata\Memory as ModelsMetaData;

$eventsManager = new EventsManager();

$di = new DI();

$connection = new Connection(array(
    "host"     => "localhost",
    "username" => "root",
    "password" => "",
    "dbname"   => "phalcon_test",
));

$connection->setEventsManager($eventsManager);

$eventsManager->attach('db',
    function ($event, $connection) {
        switch ($event->getType()) {
            case 'beforeQuery':
                echo $connection->getSqlStatement(), "\n";
                break;
        }
    }
);

$modelsManager = new ModelsManager();
$modelsManager->setDi($di);
$di['db'] = $connection;
$di['modelsManager'] = $modelsManager;
$di['modelsMetadata'] = new ModelsMetadata();

class Robots extends Model
{

}

class RobotsParts extends Model
{

}

$modelsManager->registerNamespaceAlias('s', 'Some');

$phql = "SELECT * FROM s:Robots INNER JOIN s:RobotsParts";
$query = $modelsManager->executeQuery($phql);
```
